posts/models.py

```diff
@@ -17,3 +17,18 @@ def original_or_self(self):
         while p.original_post:
             p = p.original_post
         return p
+
+    def __str__(self):
+        return self.content
+
+class Hashtag(models.Model):
+    name = models.CharField(max_length=255)
+    posts = models.ManyToManyField(Post)
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Maybe @juanecabellob could delete the current ones for this feature and simply push the changes in the migration. I don't think it's necessary to have a new migration, since it's not merged yet.